libs/user_extensions/src/TTAlpsObjetcsManager.cpp

```diff
@@ -0,0 +1,84 @@
+#include "TTAlpsObjectsManager.hpp"
+
+#include "ConfigManager.hpp"
+#include "Logger.hpp"
+
+using namespace std;
+
+TTAlpsObjectsManager::TTAlpsObjectsManager() {
+  auto& config = ConfigManager::GetInstance();
+}
+
+void TTAlpsObjectsManager::InsertLooseMuonsMatchedCollections(shared_ptr<Event> event) {
+
+  shared_ptr<PhysicsObjects> looseMuonsDRMatch = asNanoEvent(event)->GetDRMatchedMuons(0.1);
```

Comment:
I'm wondering if it's a good idea to keep these together, since later we will just pick one matching method, so others would be computed for no good reason.